[2025-07-08T04:33:22.614+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-08T04:33:22.683+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-07T07:58:00+00:00 [queued]>
[2025-07-08T04:33:22.714+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-07T07:58:00+00:00 [queued]>
[2025-07-08T04:33:22.716+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-07-08T04:33:22.778+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-07-07 07:58:00+00:00
[2025-07-08T04:33:22.812+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=109) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-08T04:33:22.818+0000] {standard_task_runner.py:72} INFO - Started process 110 to run task
[2025-07-08T04:33:22.834+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'scheduled__2025-07-07T07:58:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpo72s1a6x']
[2025-07-08T04:33:22.842+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask run_dbt_trusted
[2025-07-08T04:33:23.249+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-07T07:58:00+00:00 [running]> on host 5e24963c1987
[2025-07-08T04:33:24.051+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T07:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-07T07:58:00+00:00'
[2025-07-08T04:33:24.059+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-08T04:33:24.159+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-08T04:33:24.172+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-07-08T04:33:24.283+0000] {subprocess.py:86} INFO - Output:
[2025-07-08T04:33:24.304+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-07-08T04:33:24.308+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-07-08T04:33:35.537+0000] {subprocess.py:93} INFO - [0m04:33:35  Running with dbt=1.8.0
[2025-07-08T04:33:37.841+0000] {subprocess.py:93} INFO - [0m04:33:37  Registered adapter: postgres=1.8.2
[2025-07-08T04:33:44.156+0000] {subprocess.py:93} INFO - [0m04:33:44  Found 9 models, 10 sources, 428 macros
[2025-07-08T04:33:44.170+0000] {subprocess.py:93} INFO - [0m04:33:44
[2025-07-08T04:33:44.861+0000] {subprocess.py:93} INFO - [0m04:33:44  Concurrency: 1 threads (target='dev')
[2025-07-08T04:33:44.864+0000] {subprocess.py:93} INFO - [0m04:33:44
[2025-07-08T04:33:44.913+0000] {subprocess.py:93} INFO - [0m04:33:44  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-07-08T04:33:45.631+0000] {subprocess.py:93} INFO - [0m04:33:45  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 840[0m in 0.71s]
[2025-07-08T04:33:45.643+0000] {subprocess.py:93} INFO - [0m04:33:45  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-07-08T04:33:45.894+0000] {subprocess.py:93} INFO - [0m04:33:45  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 840[0m in 0.24s]
[2025-07-08T04:33:45.905+0000] {subprocess.py:93} INFO - [0m04:33:45  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-07-08T04:33:46.203+0000] {subprocess.py:93} INFO - [0m04:33:46  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 840[0m in 0.29s]
[2025-07-08T04:33:46.221+0000] {subprocess.py:93} INFO - [0m04:33:46  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-07-08T04:33:46.554+0000] {subprocess.py:93} INFO - [0m04:33:46  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 840[0m in 0.33s]
[2025-07-08T04:33:46.618+0000] {subprocess.py:93} INFO - [0m04:33:46
[2025-07-08T04:33:46.621+0000] {subprocess.py:93} INFO - [0m04:33:46  Finished running 4 table models in 0 hours 0 minutes and 2.44 seconds (2.44s).
[2025-07-08T04:33:46.948+0000] {subprocess.py:93} INFO - [0m04:33:46
[2025-07-08T04:33:46.953+0000] {subprocess.py:93} INFO - [0m04:33:46  [32mCompleted successfully[0m
[2025-07-08T04:33:46.956+0000] {subprocess.py:93} INFO - [0m04:33:46
[2025-07-08T04:33:46.959+0000] {subprocess.py:93} INFO - [0m04:33:46  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-07-08T04:33:48.673+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-08T04:33:48.828+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-08T04:33:48.840+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=scheduled__2025-07-07T07:58:00+00:00, execution_date=20250707T075800, start_date=20250708T043322, end_date=20250708T043348
[2025-07-08T04:33:48.972+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-08T04:33:49.037+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-08T04:33:49.044+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
